package.json

```diff
@@ -23,29 +23,29 @@
   "author": "Tiny Technologies",
   "license": "MIT",
   "devDependencies": {
-    "@ephox/agar": "^7.4.1",
-    "@ephox/bedrock-client": "^13.0.0",
-    "@ephox/bedrock-server": "^13.3.0",
+    "@ephox/agar": "^8.0.1",
+    "@ephox/bedrock-client": "^14.1.1",
+    "@ephox/bedrock-server": "^15.0.0-alpha.4",
```

Comment:
Not sure we want to pull in an alpha here?

Reply:
Downgraded in [this commit](https://github.com/tinymce/tinymce-webcomponent/pull/60/commits/b902c809a302ef0651e645529b8280e3e8dd10aa) and verified builds.